### PR TITLE
PLAT-55: Add coverage report to PRs

### PR DESCRIPTION
**Pull request by dondrzzy**
## JIRA TICKET NAME
[PLAT-55: Add coverage report to PRs ](https://issues.openmrs.org/browse/PLAT-55)

## Description of what I changed

- Updated ReadMe.md file

## Issue I worked on
[PLAT-55](https://issues.openmrs.org/browse/PLAT-55)

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Review by Rhotimee — APPROVED** [review]
LGTM

**Review by emmabaye — APPROVED** [review]
LGTM

**Review by Daramola98 — APPROVED** [review]
LGTM

**Review by muhozi — APPROVED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ script:
   - npm install coveralls
   - npm test -- -u --coverage
 after_success:
-  - npm run coverage
+  - COVERALLS_REPO_TOKEN=olYi6pj3rhE6JWKlOdLq3siCyVWxddv2E npm run coveralls
```
Comment (emmabaye) [inline .travis.yml]: Why leave such sensitive information in a public repository? This should be included in the CI environment settings.

**Review by emmabaye — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (dondrzzy) [reply .travis.yml]: Thank you @emmabaye, great attention to detail. I hope you also put up a ticket to remove it from https://github.com/openmrs/openmrs-ocl-client/blob/master/.coveralls.yml. Let me have someone with access to add the token to the env variables.

Comment (emmabaye) [reply .travis.yml]: Yes, it should be considered. I wasn't part of the team that started the project. I didn't realise it was there. It's not right. We have to adhere to best practices.

Comment (dondrzzy) [reply .travis.yml]: Addressed

**Review by emmabaye — APPROVED** [review]
LGTM

**Pull request merged**